app/assets/stylesheets/custom.css

```diff
@@ -25,12 +25,33 @@ body.app {
   color: #11223f;
 }
 
+@media (min-width: 768px) {
+  body.app {
```

Comment:
Wieso geht es für Desktop auf die body Klasse und bei mobil auf eine eigene Toggle Klasse?

Reply:
bei desktop is die sidebar in geschlossenem zustand 104px breit zu sehen immer, daher muss der content bereich dahinter, also der body, um diese breite verschoben werden nach rechts.

bei mobile is die sidebar in geschlossenem zustand weg und der hamburger (sidebarToggleTop) ganz links. sobald die sidebar aufgeht würde sie über dem hamburger legen, somit muss das beim aufgehen nach rechts verschoben werden (und wenn geschlossen wieder 0).

um das zu realisieren musste ich die selektoren so wählen.

Reply:
okay verstehe